terraform/terraform_spec.rb

```diff
@@ -18,7 +18,7 @@
   it "has the expected Terraform version" do
     expect(
       command("terraform version").stdout
-    ).to include("Terraform v0.7.3")
+    ).to include("Terraform v0.7.5")
```

Comment:
Is is actually version 0.7.5? I can see that we checkout the project from a branch. Was it based on 0.7.5 or latest? How can we check that ?

If it is based on v0.7.5 shall we add dev suffix or some other indication that this version is a custom one?